src/Invictus.Testing.Tests.Integration/LogicAppTriggerNotFoundExceptionTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using Xunit;
+
+namespace Invictus.Testing.Tests.Integration
+{
+    public class LogicAppTriggerNotFoundExceptionTests
+    {
```

Comment:
I would add a simple test that creates the exception with all logic app info and we verify that the properties are assigned as expected, without (de)serialization

Reply:
Maybe add majority of these tests here to our base `LogicAppException` as well?